### ISSUE TITLE
Touch ID popup incorrectly appearing after Vault timeout / lock

### ISSUE THREAD

**Issue by Jonyder** [bitwarden/clients#8640]
### Steps To Reproduce

Hi,
On my MacBook, I've enabled 'Ask for Touch ID on app start' in the Bitwarden settings.
To try to address the issue described below, I've removed and re-installed Bitwarden, which did not make any difference.

### Expected Result

I'm expecting that the MacOS Touch ID pop-up only appears automatically when I start the Bitwarden app, as the name of the setting clearly describes.

### Actual Result

The Touch ID pop-up appears incorrectly:
1. When the Vault timeout is reached, I've set this to 5 minutes
2. When 'Lock all vaults' is used.

### Operating System

macOS

### Operating System Version

14.4.1

### Installation method

Mac App Store

### Build Version

2024.3.0 (21897)

### Issue Tracking Info

- [X] I understand that work is tracked outside of GitHub. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by SamFrank234** [comment]
I agree this could use clarification. I think the setting should either be renamed (to something like 'Ask for Touch ID to unlock vault'), or the functionality should be modified so that the Touch ID pop up only occurs on startup.

**Comment by Krychaz** [comment]
Hello there, 

This is expected behaviour for this setting. I agree that it might sound confusing so I will suggest changing the name of it to our team. Biometric Unlock can only be used to unlock a vault in locked state. Therefore if Touch ID is enabled, and your vault is locked, Touch ID will be offered every time.

**Comment by Jonyder** [comment]
> Hello there,
> 
> This is expected behaviour for this setting. I agree that it might sound confusing so I will suggest changing the name of it to our team. Biometric Unlock can only be used to unlock a vault in locked state. Therefore if Touch ID is enabled, and your vault is locked, Touch ID will be offered every time.

Thanks for explaining! 
To me however the intended functionality doesn't make sense. Who would want to lock and immediately want to unlock the vault again. Or who sets a vault lock timeout and wants to unlock immediately when the lock timeout is reached? 

Was this a registered feature request that users voted on?
Who wants this setting 'Ask for Touch ID to unlock vault' to work in the way you describe?

**Comment by Krychaz** [comment]
Hello,

Perhaps this proposed change by @wearsshoes in #10201 would address this issue?

> While I think the bug can be resolved without changing intended program behavior, I've submitted this proposed UI fix as a feature request, since I think that changing the program behavior will reduce confusion: https://community.bitwarden.com/t/macos-desktop-change-ask-for-touch-id-on-app-start-flag-to-prompt-for-touch-id-when-activated/71360

**Comment by closebot-bw** [comment]
⚠️ **Stale Issue Notice**

This issue has been automatically marked as stale due to inactivity. It will be closed in **2 weeks** (October 15, 2025) if no further activity occurs.

If this issue is still relevant and you would like to keep it open, please:
- Comment on this issue to show continued interest
- Provide any additional information or updates
- Confirm that the issue still exists in the latest version

Thank you for your contribution to this project! 🙏

**Comment by closebot-bw** [comment]
🔒 **Issue Closed Due to Inactivity**

This issue has been automatically closed due to lack of activity for an extended period. We periodically review and close inactive issues to help maintain our issue tracker and focus on current priorities.

**If this issue is still relevant:**
- Please create a new issue with updated information
- Include steps to reproduce the problem if it's a bug report
- Mention if this issue still occurs in the latest version

Thank you for your contribution to this project. Your feedback helps us improve! 🙏
